CMakeLists.txt

```diff
@@ -85,9 +85,10 @@ endif()
 #
 find_package(PkgConfig)
 pkg_check_modules(GST
+	REQUIRED
     gstreamer-1.0>=1.14
     gstreamer-video-1.0>=1.14
-    gstreamer-gl-1.0>=1.14
+    gstreamer-gl-1.0>=1.14	
```

Comment:
Thanks for that finding - we missed that. But could you please remove extra spaces at the end of line?